tools/azure-sdk-tools/packaging_tools/auto_package_codegen.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import json
+import logging
+import os
+from pathlib import Path
+
+from azure_devtools.ci_tools.git_tools import get_diff_file_list
+from .common_utils import create_package, change_log_generate, extract_breaking_change
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def main(generate_input, generate_output):
+    with open(generate_input, "r") as reader:
+        data = json.load(reader)
+
+    sdk_folder = "."
+    result = {"packages": []}
+    for package in data.values():
+        package_name = package["packageName"]
+        # Changelog
+        last_version = ["first release"]
+        if 'azure-mgmt-' in package_name:
+            md_output = change_log_generate(package_name, last_version)
+        else:
+            md_output = "data-plan skip changelog generation temporarily"
+        package["changelog"] = {
+            "content": md_output,
+            "hasBreakingChange": "Breaking changes" in md_output,
+            "breakingChangeItems": extract_breaking_change(md_output),
+        }
+
```

Comment:
```suggestion
        package["version"] = last_version[-1]
```
Please keep this in output json since we has used it in another pipeline.

Reply:
done